static/css_all.css

```diff
@@ -8093,3 +8093,41 @@ fieldset[disabled] .radio,
 .fa-wheelchair:before{content:"\f193"}
 .fa-vimeo-square:before{content:"\f194"}
 .fa-turkish-lira:before,.fa-try:before{content:"\f195"}
+
```

Comment:
@ttseng I actually should be refactoring this file out of the repo, it's supposed to get generated dynamically.  Your custom CSS should all go your separate file and that file should be wired in with Flask Assets.  I can help do that once you've opened a branch on the main repo.